index.html

```diff
@@ -0,0 +1,229 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+
+    <!-- CSS -->
+    <link rel="stylesheet" href="css/style.css">
+
+    <!-- Font Awesome -->
+    <script src="https://kit.fontawesome.com/2f09744991.js" crossorigin="anonymous"></script>
+    <title>Learn Web Development</title>
+  </head>
+  <body>
+    <!-- Introduction -->
+    <section id="intro" class="pb-4">
+        <header>
+            <ul class="header m-0">
+                <li class="header-item"><a class="text-white" href="#"><i class="fa-brands fa-facebook"></i></a></li>
+                <li class="header-item"><a class="text-white" href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                <li class="header-item">English</li>
+                <li class="header-item">My Page</li>
+                <li class="header-item">Logout</li>
+            </ul>
+        </header>
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col p-0">
+                    <nav class="navbar">
+
+                        <a href="index.html"><img class="logo" alt="Logo" src="images/web-development-logo.jpg"></a>
+                
+                        <ul class="nav-options-ul">
+                            <li class="nav-options-ul-li"><a class="nav-options-ul-li-a" href="index.html">Home</a></li>
+                            <li class="nav-options-ul-li"><a class="nav-options-ul-li-a" href="about.html">About</a></li>
+                            <li class="nav-options-ul-li"><a class="nav-options-ul-li-a" href="#main-program">Courses</a></li>
+                            <li class="nav-options-ul-li"><a class="nav-options-ul-li-a" href="#join">Join</a></li>
+                            <li class="nav-options-ul-li"><a class="nav-options-ul-li-a" href="#sponser">Sponser</a></li>
+                            <li class="nav-options-ul-li"><a class="nav-options-ul-li-a" href="#news">News</a></li>
+                        </ul>
```

Comment:
- [ ] As the last code reviewer said, please 🙏🏽 follow the guides of the original design. The navbar links on the home page are bolder than the links on the about page kindly use the same font style on both pages the home page and the about page must be similar to the original design and please 🙏🏽 correct the color when hovering the links in the about page it use red instead of blue.

Reply:
@franclobo Please check the code and output again. I have used same font and size. Thanks